apps/prairielearn/src/tests/bonusPoints.test.ts

```diff
@@ -50,15 +51,14 @@ describe('Exam assessment with bonus points', function () {
   });
 
   step('submit an answer to the first question', async () => {
-    const form = {
-      __action: 'grade',
-      __csrf_token: context.__csrf_token,
-      __variant_id: context.__variant_id,
-      s: '75', // To get 75% of the question
-    };
-    const response = await helperClient.fetchCheerio(context.question1Url, {
+    const response = await fetch(context.question1Url, {
```

Comment:
This will cause a small change in behaviour, in that it will await for the header, but not necessarily the full response. I don't think this is a problem, just making sure you considered it. Same for the other places where fetchCheerio was replaced with fetch.

Reply:
This is fine, I think. We already do this in many other places in the tests.